### ISSUE TITLE
BUG: Primary key sign is not present in both columns of composite key

### ISSUE THREAD

**Issue by maktsakalis** [beekeeper-studio/beekeeper-studio#464]
**Describe the bug**
When creating a composite primary key, the primary key sign 
![3](https://user-images.githubusercontent.com/1318089/101268007-6d4f3a00-3767-11eb-9689-fcd0c47f04a3.png)
is not present in both columns when i re-open the tab.

**To Reproduce**

1. User has to create a collection of value types for an entity via Hibernate annotations (in my case the entity's name is "**Friend**",  "**friend_nickname**" with columns: **nickname** and **friend_id**)

2. Afterwards user runs an alter statement in order to create a composite key: 
alter table friend_nickname add primary key (friend_id, nickname);

3. After the statement has been executed user expects both columns in friend_nickname table to have the sign of primary key

4. Even though the table is altered and both columns are considered as primary key, the sign of primary key is present only in foreign key's column "friend_id"

**Expected behavior**
The sign of primary key was expected to be in both **nickname** and **friend_id** columns

**Screenshots / Gifs**
![1](https://user-images.githubusercontent.com/1318089/101267993-455fd680-3767-11eb-8cc7-04d36c5e3c2b.png)
![2](https://user-images.githubusercontent.com/1318089/101267995-47299a00-3767-11eb-96a7-418f2239e36f.png)

**Version Information (please complete the following information):**
 - OS: Manjaro Linux Nibia 20.2
 - App Version: 1.8.11

**Additional context**
AUR package: https://aur.archlinux.org/packages/beekeeper-studio-bin

**Comment by rathboma** [comment]
Yes this is a known bug and is to make sure we don't allow editing for composite keys until we've finished that feature.

Basically it treats the table as having 1 or 0 keys.

**Comment by maktsakalis** [comment]
Thank you very much for your reply and your excellent work!
